io.github.pieterdd.RcloneShuttle.yaml

```diff
@@ -0,0 +1,39 @@
+id: io.github.pieterdd.RcloneShuttle
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: rclone-shuttle
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--device=dri"
+- "--env=G_MESSAGES_DEBUG=none"
+- "--env=RUST_BACKTRACE=1"
+- "--filesystem=host"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/rust-flatpak/cargo"
+modules:
+  - name: rust-flatpak
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/rust-stable/bin/cargo --offline fetch --manifest-path Cargo.toml --verbose
+    - /usr/lib/sdk/rust-stable/bin/cargo --offline build --release --verbose
+    - mkdir -p /app/bin
+    - install -Dm755 ./target/release/rclone-shuttle /app/bin/rclone-shuttle
+    - ln -s /run/host/usr/bin/rclone /app/bin/rclone
+    - install -Dm644 -t /app/share/icons/hicolor/512x512/apps/ meta/io.github.pieterdd.RcloneShuttle.png 
+    - install -Dm644 -t /app/share/applications/ meta/io.github.pieterdd.RcloneShuttle.desktop
+    - install -Dm644 -t /app/share/metainfo/ meta/io.github.pieterdd.RcloneShuttle.metainfo.xml
+    sources:
+    - type: dir
+      path: .
```

Comment:
and how to you think it find the source code here?

Reply:
I'm new to this, I only know the manual said I should add my manifest here. Should I mirror all source code here? Because that might get tricky to keep in sync with the real repo.

Reply:
> Should I mirror all source code here?

No. There are literally THOUSANDS of package to look at for inspiration.

Reply:
tl;dr you specify a source, either a tarball or a git repository.

the basic example in the doc show how to do that with a tarball

https://docs.flathub.org/docs/for-app-authors/requirements